crud-routing/burger.router.js

```diff
@@ -0,0 +1,81 @@
+const burgerRouter = require('express').Router();
+const bodyParser = require('body-parser');
+const fs = require('fs');
+require('dotenv').config();
+const menuFile = process.env.MENU_FILE;
+
+burgerRouter.use(bodyParser.json());
```

Comment:
use body parser once in main file don't need to use it in all routes also body-parses package is not needed i thought we had discussed this.

Reply:
remove body-parser